lua/typst-preview/fetch.lua

```diff
@@ -209,14 +209,22 @@ function M.fetch(quiet, callback)
           .. utils.get_data_path()
       )
     end
-    local record, err = io.open(record_path, 'w')
-    if record == nil then
-      error("Can't open record file!: " .. err)
-    end
+    local record_required = true
     for _, bin in pairs(M.bins_to_fetch()) do
-      record:write(bin.url .. '\n')
+      if config.opts.dependencies_bin[bin.name] ~= nil then
```

Comment:
Wouldn't this mark record_required as false when one of the dependencies_bin doesn't need to be downloaded but the other one does?

Reply:
ah true, my bad